lib/node_modules/@stdlib/blas/base/snrm2/lib/ndarray.js

```diff
@@ -31,21 +31,19 @@ var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
 /**
 * Computes the L2-norm of a single-precision floating-point vector.
 *
-* @param {PositiveInteger} N - number of values over which to compute the L2-norm
+* @param {PositiveInteger} N - number of values
 * @param {Float32Array} x - input array
 * @param {integer} stride - stride length
 * @param {NonNegativeInteger} offset - starting index
-* @returns {number} L2-norm
+* @returns {number} output value
```

Comment:
```suggestion
* @returns {number} L2-norm
```

We leave this description intact in elsewhere, so we should be consistent here.